packages/google_identity_services_web/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 0.3.3
 
-* Move all the JavaScript types to extend `JSObject`.
+* Migrates all the JavaScript types to the latest syntax.
```

Comment:
Yeah, but that doesn't tell the user WHAT changed. The fact that these types are now `JSObject` *is* the user-visible change.

Reply:
Users never see these objects, they don't care about the underlying representation, but let me revert.